File: findMaxNumber.java
package springIntro;

import java.util.Scanner;

public class findMaxNumber {

	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);
		int number = 0;
		int i = 3;
		int max = number;
		while(i > 0) {
			number = in.nextInt();
			if(max > number) {
				
			}
			else {	
				max = number;
			}
			i--;
		}
		System.out.println(max);
	}
}

File: findMaxNumber.cpp
#include<iostream>
using namespace std;
int main(){
	int a, b, c;
	cout << "Enter three number = ";
	cin >> a >> b >> c;
	int max;
	max = a;
	if(max < b)
		max = b;
		if(max < c)
			max = c;
	else
		if(max < c)
			max = c;
			
	cout << "Max number = " << max;
	return 0;
}
